gitpod-io/website
# ✨ Revamp `/discover/opensource`

## Description



Updates to the Gitpod for Open Source page ✨

## Related Issue(s)



Fixes [MKWEB-272 : Update /discover/opensource page](https://linear.app/gitpod/issue/MKWEB-272/update-discoveropensource-page)

## How to test



- **Open** [**preview**](https://deploy-preview-3975--gitpod-kumquat.netlify.app/discover/opensource) and **see** :eyes:

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing


## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.


</details>


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3975"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
